model_api/cpp/models/src/anomaly_model.cpp

```diff
@@ -0,0 +1,230 @@
+/*
+  Copyright (C) 2020-2023 Intel Corporation
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+#include "models/anomaly_model.h"
+
+#include <memory>
+#include <openvino/core/any.hpp>
+#include <openvino/core/model.hpp>
+#include <ostream>
+
+#include "models/image_model.h"
+#include "models/input_data.h"
+#include "models/internal_model_data.h"
+#include "models/results.h"
+#include "utils/slog.hpp"
+
+std::string AnomalyModel::ModelType = "AnomalyDetection";
+
+/// @brief Initializes the model from the given configuration
+/// @param top_priority  Uses this as the primary source for setting the parameters
+/// @param mid_priority Fallback source for setting the parameters
+void AnomalyModel::init_from_config(const ov::AnyMap &top_priority, const ov::AnyMap &mid_priority)
+{
+  mean_values = get_from_any_maps("mean_values", top_priority, mid_priority, mean_values);
```

Comment:
You still use 4 spaces sometimes

Reply:
I have this clangd plugin in my ide. Not sure if it is working correctly.

Reply:
There are still lots of them. One example is `  class Model;` in
```cpp
namespace ov
{
  class Model;
} // namespace ov
```